<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1221</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter eleve</string>
     </attribute>
     <widget class="QLineEdit" name="line_nom">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_prenom">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date de naissance</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_n">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_classe">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Classe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>360</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Genre</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_genre">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>360</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Homme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Femme</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="line_tel">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>90</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Num tel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_adresse">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>160</y>
        <width>211</width>
        <height>87</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>190</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>290</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Niveau</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_niveau">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>290</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1ere</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2eme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3eme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4eme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5eme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6eme</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>370</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pension</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_pension">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>370</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Externe</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Demi-pension</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pension-complete</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>450</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion eleves</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>571</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Supprimer_E">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Modifier_E">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>170</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Classe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>120</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date de naissance</string>
      </property>
     </widget>
     <widget class="QComboBox" name="genre_edit">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>210</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Homme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Femme</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="prenom_edit">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_edit">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>20</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_edit">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>110</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>210</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Genre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>70</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="classe_edit">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>310</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QComboBox" name="niveau_edit">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>370</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1ere</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2eme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3eme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4eme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5eme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6eme</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="adresse_edit">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>290</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="pension_edit">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>420</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Externe</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Demi-pension</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pension-complete</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>250</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Num tel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>370</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Niveau</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tel_edit">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>250</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>420</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pension</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_C">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>370</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Croissant</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_D">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>400</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Decroissant</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_tri">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>380</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Par Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Par Niveau</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Par Classe</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="Tri_E">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>450</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1223</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
